2-mandatory/3-stocks.js

```diff
@@ -48,7 +59,13 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
```

Comment:
This looks good as well.
One small comment: Maybe `item` is not the best variable name here, something like `prices` will give the reader of the code a bit more information.